components/TimeBarChart.vue

```diff
@@ -75,15 +110,15 @@ export default {
       if (this.dataKind === 'transition') {
         return {
           lText: `${this.chartData.slice(-1)[0].transition.toLocaleString()}`,
-          sText: `実績値（前日比：${this.displayTransitionRatio} ${this.unit}）`,
+          sText: `${this.$t('実績値')}（${this.$t('前日比')}：${this.displayTransitionRatio} ${this.unit}）`,
```

Comment:
若干i18nから外れますが、全角コロンの部分を半角コロン+スペースに修正お願いいたします
```suggestion
          sText: `${this.$t('実績値')}（${this.$t('前日比')}: ${this.displayTransitionRatio} ${this.unit}）`,
```

Reply:
対応しました